package.json

```diff
@@ -54,7 +54,7 @@
         "eslint-plugin-cypress": "^2.8.1",
         "eslint-plugin-flowtype": "^4.5.2",
         "eslint-plugin-formatjs": "^1.5.4",
-        "eslint-plugin-import": "^2.19.1",
+        "eslint-plugin-import": "^2.22.1",
```

Comment:
Was running into an eslint issue when trying to commit and push up, used recommendation from this article to be able to commit without the --no-verify
https://github.com/airbnb/javascript/issues/2331